scripts/extensions/sams/src/notifications/assets.ts

```diff
@@ -0,0 +1,98 @@
+import {getStore} from '../store';
+import {receiveAssets} from '../store/assets/actions';
+import {
+    closeAssetContentPanel,
+    queryAssetsFromCurrentSearch,
+    updateSelectedAssetIds,
+    sessionUnlock,
+    loadAssetsByIds,
+} from '../store/assets/actions';
+import {superdeskApi, samsApi} from '../../src/apis';
+import {ISAMSWebsocketEvent, LIST_ACTION} from '../interfaces';
+import {
+    getAssetSearchParams,
+    getAssetListStyle,
+    getAssetListItemIds,
+    getSelectedAssetId,
+    getSelectedAssetIds,
+} from '../store/assets/selectors';
+
+export function onAssetCreated(event: ISAMSWebsocketEvent) {
+    const store = getStore();
+    const item_id = event.detail.extra.item_id!;
+    const params = getAssetSearchParams(store?.getState());
+    const listStyle = getAssetListStyle(store?.getState());
+
+    return samsApi.assets.query(params, listStyle)
+        .then((response) => {
+            Object.keys(response._items).forEach((element: any) => {
```

Comment:
`response._items` is an `Array<IAssetItem>`, so you can simply run:

```
response._items.forEach((item: IAssetItem) => {
    // logic
})
```

It's also best to avoid the use of `any`. This makes it easier for developers to understand the code, and allows the compiler to pick up on problems.